mautic/mautic
# Support assigned  field's list choices for companies in  forms

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/5556
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

This bugfix/feature has been sponsored by @Webmecanik

[//]: # ( Required: )
#### Description:
Options in forms "Use assigned contact field's list choices"  pre-fill options from contact fields. This PR added support also from company fields

![image](https://user-images.githubusercontent.com/462477/34921952-6750ca8e-f989-11e7-952a-ac78a489ad11.png)


[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1.  Create custom multi select custom fields with company object
2.  Create form and choose "select field" and go to properties tab and  set Yes "Use assigned contact field's list choices". 
3. Save and see select field wasn't pre-filled

#### Steps to test this PR:
1.  Apply PR and repeat steps to reproduce
2. See If options was pre-filled from contact custom field like on this video http://youtu.be/_2ku1gZXKyQ?hd=1
